packages/liferay-npm-bundler-preset-portal/config.json

```diff
@@ -110,5 +110,6 @@
 				"metal-state": ">=2.16.0"
 			}
 		}
-	}
+	},
+	"output": "classes/META-INF/resources"
```

Comment:
IMO this should go in the preset because it's specific of portal's build and may break customers' projects.

Reply:
@izaera Ok, thanks. I'll remove it then.